doc/nrf/known_issues.rst

```diff
@@ -225,6 +225,144 @@ Missing :file:`CMakeLists.txt`
   You can `download this file <nRF52820 CMakeLists.txt_>`_ from the upstream Zephyr repository.
   After you add it, the file is automatically included by the build system.
 
+  Bugfixes
```

Comment:
This file is for known issues, thus problems users need to know about when using a specific release.
Fixes should go into the changelog for the SoftDevice Controller (or maybe MPSL): https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/softdevice_controller/CHANGELOG.html

Reply:
Hi, this should be ready for re-review.